web/src/actions/auth.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+
+const API_URL = 'http://localhost:8000';
```

Comment:
In f6a7cc69, I added a webpack config to replace instances of `process.env.API_URL` with the actual value of that environment variable at build time, so this should work if it's changed to:

```js
const API_URL = process.env.API_URL;
```

The deploy scripts that CircleCI uses are setting the `API_URL` appropriately, based on which environment it's going to.

Feel free to push back on this as a global approach (maybe we should only inject environment variables into something like an `env.js` file?), but we definitely need some kind of solution for setting the API URL.

Reply:
oh cool, that sounds reasonable to me -- will change it over